docs/use/kingfisher-process.rst

```diff
@@ -0,0 +1,19 @@
+Kingfisher Process
+==================
+
+Read the `Kingfisher Process <https://kingfisher-process.readthedocs.io/en/latest/>`__ documentation, which cover general usage.
+
+View Logs
+---------
+
+The logs from Process are available for the ``ocdskfs`` and ``ocdskfp`` users to read.
+
+The newest logs are at ``/var/log/kingfisher.log``.
+
+Log rotation is used, and older logs can be found at ``/var/log/kingfisher.log.1`` and other compressed files like ``/var/log/kingfisher.log.2.gz``.
```

Comment:
Make into one paragraph.

Maybe tag this paragraph with `**Location:**` and the previous paragraph with `**Access:**`.